lib/reports/brexit_subscribers_report.rb

```diff
@@ -4,16 +4,18 @@ class Reports::BrexitSubscribersReport
   CSV_HEADERS = %w[title slug tags subscribed unsubscribed immediately daily weekly].freeze
   attr_reader :date
 
-  def self.call(*args)
-    new(args).export_csv
+  def call
+    export_csv
   end
```

Comment:
```suggestion
  def self.call(*args)
    new(*args).call
  end
  
  def call
    subscriber_lists = date.nil? ? brexit_lists : brexit_lists_before_date

    CSV($stdout, headers: CSV_HEADERS, write_headers: true) do |csv|
      subscriber_lists.each do |list|
        csv << row_data(list)
      end
  end
```

This would make this a consistent interface with the common use of `.call` in this app. And would mean you could skip the new(date) when calling. An example this is: https://github.com/alphagov/email-alert-api/blob/cd0330133f6ead2f9d1db02e069eff3c6d9ccaaf/app/builders/content_change_email_builder.rb#L6-L13

Reply:
Ah ok, I misunderstood your comments on the previous pr! I thought your comment there was to remove the `self.call` method altogether and replace it with an instance method, but I see what you mean was to rename the export_csv task ==> call. Thanks for explaining.